demoapps/simplest/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Nit: Is this already ignored by the root .gitignore?

Reply:
Yes maybe, it's an auto-generated file, I will try to remove it

Reply:
Removed